src/shared/components/community/create-community.tsx

```diff
@@ -48,6 +50,7 @@ export class CreateCommunity extends Component<any, CreateCommunityState> {
   render() {
     return (
       <div className="container-lg">
+        {!UserService.Instance.myUserInfo && <Redirect to="/login" />}
```

Comment:
This logic is already taken care of with the `AuthGaurd` component.

Reply:
This is not part of my commit

Reply:
It must have snuck in during the merge.